<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatMasterWidget</class>
 <widget class="QWidget" name="ChatMasterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Veyon Chat - Master</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/chat.qrc">
    <normaloff>:/chat/icons/chat.png</normaloff>:/chat/icons/chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLabel" name="clientsLabel">
         <property name="text">
          <string>Connected Clients</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="clientList">
         <property name="minimumWidth">
          <number>200</number>
         </property>
         <property name="maximumWidth">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QLabel" name="chatLabel">
         <property name="text">
          <string>Chat</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="chatDisplay">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="inputPanel">
         <layout class="QHBoxLayout" name="inputLayout">
          <item>
           <widget class="QComboBox" name="quickReplies">
            <property name="toolTip">
             <string>Quick Reply Templates</string>
            </property>
            <item>
             <property name="text">
              <string>Quick Replies...</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Please raise your hand if you have questions</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Good work! Keep it up.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Please focus on your work</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Break time in 5 minutes</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="priorityCombo">
            <property name="toolTip">
             <string>Message Priority</string>
            </property>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Urgent</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Announcement</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="messagePanel">
         <layout class="QHBoxLayout" name="messageLayout">
          <item>
           <widget class="QLineEdit" name="messageInput">
            <property name="placeholderText">
             <string>Type your message here...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="text">
             <string>Send</string>
            </property>
            <property name="toolTip">
             <string>Send message (Ctrl+Enter)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="buttonPanel">
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear Chat</string>
            </property>
            <property name="toolTip">
             <string>Clear chat history for selected client</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="globalButton">
            <property name="text">
             <string>Global Broadcast</string>
            </property>
            <property name="toolTip">
             <string>Send message to all clients</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="text">
             <string>Ready</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/chat.qrc"/>
 </resources>
 <connections/>
</ui>
